packages/sdk/src/utils/loadEnvFileAndPrint.ts

```diff
@@ -0,0 +1,12 @@
+import { tryLoadEnvs } from './tryLoadEnvs'
+import { getEnvPaths } from './getEnvPaths'
+
+/**
+ * Read .env file only if next to schema.prisma
+ * .env found: print to console it's relative path
+ */
+export function loadEnvFileAndPrint(schemaPath?: string) {
+  const envPaths = getEnvPaths(schemaPath)
+  const envData = tryLoadEnvs(envPaths, { conflictCheck: 'error' })
+  envData && envData.message && console.info(envData.message)
```

Comment:
Do we need to re-add https://github.com/prisma/prisma/pull/11521/files#diff-f68aae81afb0da1362bf689b66647746ff60979dcf684bd4bd4ff7d6297100f6L78 here?